02-primitives/01-numbers/README.md

```diff
@@ -0,0 +1,7 @@
+# Numbers - Calculate the area of shapes
```

Comment:
Please remove all generated files (`README.md`, `exercise_test.go`) from the repo: we only need the templates. Make sure to call `make realclean~ before pushing!

Reply:
I used that command but maybe I created some changes after...